README.md

```diff
@@ -9,31 +9,43 @@ The main functionalities:
 ## Tech/Frameworks
 - **Front-End:** HTML5, CSS3, jQuery3/JavaScript
 - **Back-End:** Python `2.7.x`, Django `1.11.x`
-- **Database:** MySQL `5.6.35` on AWS RDS
+- **Database:** PostgreSQL `9.6` on AWS RDS
 
 ## Humans
 #### Lead Developers
-- **Jamie Polintan** - @jamiepoli - Software Lead, Security
 - **Seung Won [Tom] Lee** - @leesw98 - Team Lead, Front-End + Back-End
 
 #### Developers
 - **Ying Ying Choi** - @ying-choi - Developer, DB
-- **Nick Chim** - @kotes95 - Developer, Front-End + DB
 - **Omar Tsai** - @omar2535 - Developer, Front-End + Back-End
-- **Hugo Xie** - @kaitian-xie - Developer, Front-End + Back-End
 - **Michelle Huh** - @michellehuh - Developer,Front-End + Back-End
 - **Vincent Lin** - @Csignore - Developer, Back-End + DB
-- **Eric Yang** - @ericyang97 - Developer, Back-End + DB
 - **Gaurav Vasudev** - @gauravnv - Developer, Back-End + DB
-- **Nguyen [Andy] Anh Khoi** - @gh0stl0nely - Developer, Back-End
+- **David Kim** - @yuubd - Developer, Back-End + Front-End
+- **Joon Hur** - @hurjun1995 - Developer, Back-End + Front-End
 
 
+# Installation
 
-## Installation
-Clone the repo from:
+## 1) Installing Environment
+- Install the required python version 2.7.x
+- Install `pip` according to your OS
+- Install _virtualenv_ using `pip install virtualenv`
+- Install _virtualenvwrapper_ using `pip install virtualenvwrapper-win` for Windows and `pip install virtualenvwrapper` otherwise
+- Create virtualenv and use `workon env_name` command to activate the virtualenv
+    - Check out a tutorial if you're unsure
+
+## 2) Getting the project
+
+Clone the repo:
 - **HTTPS** `git clone https://github.com/CodeTheChangeUBC/ReBOOT.git`
 - **SSH** `ssh git@github.com:CodeTheChangeUBC/ReBOOT.git`
 
+Install Requirements:
+- `cd project_directory`
```

Comment:
Remove and add `brew install rabbitmq` (one for windows too)

Reply:
Also postgres app for mac and idk what for window

Reply:
Got it!